addons/account/models/partner.py

```diff
@@ -105,6 +105,7 @@ def create(self, vals):
         zip_to = vals.get('zip_to')
         if zip_from and zip_to:
             vals['zip_from'], vals['zip_to'] = self._convert_zip_values(zip_from, zip_to)
+        self.clear_caches()
```

Comment:
We should only do that if the new fiscal position has auto_apply==True.

Reply:
of course, I'll change it.

Reply:
If someone sets the auto_apply to default = True then this invalidation might be wrong and the developer won't see it. It's identical to the other side, I think it is better to favor robustness with a tiny loss of performance (compared to the gain of the current pr)